app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -37,7 +37,8 @@ def initialize(http_response, config, liveness_checking_enabled = false)
         #    document type
         #    bar code attention
         def successful_result?
```

Comment:
I'm not quite sure I see how this is connected to [LG-12370](https://cm-jira.usa.gov/browse/LG-12370). Do you mind clarifying?

Reply:
@night-jellyfish  - it's not a solution to the issue ... as is, the code was a bit challenging to follow and this is a simplification refactor made while debugging